.mailmap

```diff
@@ -1304,6 +1304,7 @@ Subhash Saurabh <subhashsaurabh419@gmail.com>
 Sudarshan Kamath <sudarshan.kamath97@gmail.com>
 Sudeep Sidhu <sudeepmanilsidhu@gmail.com> sidhu1012 <sudeepmanilsidhu@gmail.com>
 Sudhanshu Mishra <mrsud94@gmail.com>
+Sujant Kumar Kv <sujantkv@gmail.com>
```

Comment:
In your initial commit, you've managed to have two authors, which lets `bin/mailmap_check.py` fail. Second commit author is: `Sujant Kumar Kv <sujant@Sujants-MacBook-Air.local>`